activerecord/lib/active_record/railtie.rb

```diff
@@ -67,6 +68,12 @@ class Railtie < Rails::Railtie # :nodoc:
       require "active_record/base"
     end
 
+    initializer "active_record.has_many_inversing" do |app|
+      ActiveSupport.on_load(:active_record) do
+        self.has_many_inversing = app.config.active_record.has_many_inversing
```

Comment:
You don't need this intializer. The setting config initializer will take care of it.

Reply:
Fixed. CI should also pass now